packages/wdio-appium-service/src/launcher.ts

```diff
@@ -125,4 +146,18 @@ export default class AppiumLauncher {
         this.process.stdout.pipe(logStream)
         this.process.stderr.pipe(logStream)
     }
+
+    private static getAppiumCommand (moduleName = 'appium') {
```

Comment:
Let's prefix everything with underscore if it is private

```suggestion
    private static _getAppiumCommand (moduleName = 'appium') {
```

Reply:
fixed